html/user/lookup_account.php

```diff
@@ -63,6 +63,7 @@
     if (!$passwd_hash) {
         echo "<account_out>\n";
         echo "   <success/>\n";
+        echo "<id>$user->id</id>\n";
```

Comment:
Please also indent the new response line as was done to the existing line above.